packages/lit-dev-content/site/css/blog-common.css

```diff
@@ -1,62 +1,29 @@
-main {
-  background: var(--color-dark-gray);
-  line-height: 1.8em;
-  --blog-top-padding: 2em;
-}
-
-article {
-  color: white;
-  display: flex;
-  max-width: var(--content-max-width);
-  min-height: inherit;
-}
-
-#frontMatter {
-  background: var(--color-blue);
-  padding: var(--blog-top-padding) 1.5em 2em 2em;
-  flex: 0 0 14em;
-  position: relative;
-  z-index: 0;
-}
-
-#frontMatter::before {
-  position: absolute;
-  background: url('/images/texture-white.png');
-  content: '';
-  opacity: 30%;
+#splash {
+  box-shadow: rgb(0 0 0 / 20%) 0px 2px 5px 2px;
+  display: block;
+  height: 80px;
+  object-fit: cover;
   width: 100%;
-  height: 100%;
-  left: 0;
-  top: 0;
-  z-index: -1;
 }
 
-#frontMatter > h1 {
-  font-weight: 500;
-  line-height: 1.5em;
-  margin-top: 0;
-  font-size: 1.5em;
-}
-
-#blogContent {
-  padding: var(--blog-top-padding) 3em;
+article {
+  max-width: 800px;
```

Comment:
might be nice to have a site-wide CSS variable for this width

Reply:
Yeah... it's not consistent right now. The home page can get quite a lot wider, but it uses multiple columns so that seems fine. Our docs main content page is slight off 800 px because I think I tuned it for some specific break points based on the other columns. Maybe could be unified, but don't think I'll get into that for this PR.